debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java

```diff
@@ -76,7 +76,7 @@
     private final TransactionMonitor transactionMonitor;
     private final CommonConnectorConfig connectorConfig;
     private final Set<Operation> skippedOperations;
-    private final boolean neverSkip;
+    private boolean neverSkip;
```

Comment:
Why is this not any longer final?

Reply:
it was related to my first commit that added a setter method for this variable. anyway I fixedi it